packages/plus/src/components/OrderSummary/index.tsx

```diff
@@ -0,0 +1,146 @@
+import styled from '@emotion/styled'
+import { Stack, Text, UnitInput } from '@ultraviolet/ui'
+import { useMemo, useState } from 'react'
+import { NonScrollableContent } from './NonScrollableContent'
+import { OrderSummaryContext } from './Provider'
+import { ScrollableContent } from './ScrollableContent'
+import { Units } from './constants'
+import { calculatePrice } from './helpers'
+import estimateCostLocales from './locales/en'
+import type { OrderSummaryProps, TimeUnit } from './types'
+
+const Container = styled(Stack)`
+background-color: ${({ theme }) => theme.colors.neutral.backgroundWeak};
+width: 27.5rem;
+border: 1px solid ${({ theme }) => theme.colors.neutral.border};
+`
+
+const HeaderContainer = styled(Stack)`
+height: ${({ theme }) => theme.sizing[900]};
+border-bottom: 1px solid ${({ theme }) => theme.colors.neutral.border};
+padding: ${({ theme }) => theme.space[3]};
+padding-bottom: ${({ theme }) => theme.space[2]};
+`
+
+export const OrderSummary = ({
+  header,
+  hideTimeUnit = false,
+  periodOptions = Units as TimeUnit[],
+  valueUnitInput = 1,
+  unitUnitInput = 'hours',
+  items,
+  locales = estimateCostLocales,
```

Comment:
I know you took some stuff from old estimate cost but the component is now nammed OrderSummary so you might need to change the naming 😛